fieldservice_stock/models/stock_move.py

```diff
@@ -20,8 +20,11 @@ def prepare_equipment_values(self, move_line):
     def _action_done(self):
         res = super()._action_done()
         for rec in self:
-            if rec.picking_code == 'outgoing' and rec.state == 'done':
-                if rec.product_tmpl_id.create_fsm_equipment:
+            create_fsm_equipment = rec.product_tmpl_id.create_fsm_equipment
+            if rec.state == 'done' and create_fsm_equipment:
+                is_outgoing = rec.picking_code == 'outgoing'
+                can_create_fsm = rec.picking_type_id.can_create_fsm_equipment
+                if can_create_fsm or is_outgoing:
```

Comment:
@dreispt 

I think we can simplify this by dropping the picking_code. I can easily see a company  willing to create an equipment when a product is received.

```
if rec.state == 'done' and \
    rec.product_tmpl_id.create_fsm_equipment and \
    rec.picking_type_id.create_fsm_equipment:
```

Reply:
I worry about being "stable" and not breaking current installs.
Do you think that is something we could waive?

Reply:
@dreispt We can have a migration script that check the box on the picking type based on the code.